config/crds/autoscaling_v1alpha1_hvpa.yaml

```diff
@@ -214,34 +232,50 @@ spec:
                   format: int32
                   type: integer
               type: object
-            hvpaCurrentStatus:
-              description: Current information about HVPA.
+            hpaUpdatePolicy:
+              description: Current HPA UpdatePolicy set in the spec
               properties:
-                hpaWeight:
-                  format: float
-                  type: number
-                lastScaleTime:
-                  description: last time the HVPA scaled the resource; used by the
-                    autoscaler to control how often the scaling is done.
+                updateMode:
+                  description: Controls when autoscaler applies changes to the resources.
+                    The default is 'On'.
+                  type: string
+              type: object
+            hpaWeight:
+              format: float
+              type: number
+            lastError:
+              description: LastError has details of any errors that occured
+              properties:
+                description:
+                  description: Description of the error
+                  type: string
+                lastOperation:
+                  description: LastOperation is the type of operation for which error
+                    occurred
+                  type: string
+                lastUpdateTime:
+                  description: Time at which the error occurred
                   format: date-time
                   type: string
-                lastScaleType:
-                  description: the kind of scaling that was done last time
-                  properties:
-                    horizontal:
-                      type: string
-                    vertical:
-                      type: string
-                  type: object
-                overrideScaleUpStabilization:
-                  description: Override scale up stabilization window
-                  type: boolean
-                vpaWeight:
-                  format: float
-                  type: number
               type: object
+            lastScaleTime:
```

Comment:
Did you get a chance to look at the proposed structure [here](https://github.com/gardener/hvpa-controller/issues/14)?

Reply:
The implemented way already communicates whether the recommendations are `Applied` or `BlockedByUpdatePolicy` clearly. It doesn't communicate `BlockedBySpec` as of now, which would be good to have.
Will make the changes.